interpreter.py

```diff
@@ -2,8 +2,9 @@
 from rpython.rlib import jit
 from type_wrapper import String, Integer, Float, Ptr, List,\
                          Value, NoValue, NumericValue, BasicBlock
-from llvm_wrapper import *
+from llvm_objects import W_Module
 from state import State
+from llvm_wrapper import *
```

Comment:
I don't know another way around it though. I'm using all the functions defined there and writing 'llvm_wrapper.' every time I use one of them doesn't seem like a better solution to me.